community/kernel/src/main/java/org/neo4j/kernel/impl/proc/ReflectiveProcedureCompiler.java

```diff
@@ -588,6 +588,12 @@ public ProcedureSignature signature()
                             numberOfDeclaredArguments, input.length );
                 }
 
+                // Some input fields are not supported by Cypher and need to be mapped
+                for ( int i = 0; i < input.length; i++ )
```

Comment:
You need to do something similar for user-defined functions and aggregations as well

Reply:
Done